Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -262,6 +262,9 @@
     [m_openCommandEncoders removeObjectForKey:commandBuffer];
     [m_createdNotCommittedBuffers removeObject:commandBuffer];
     ++m_submittedCommandBufferCount;
+#if !PLATFORM(IOS_FAMILY_SIMULATOR)
+    [m_commitSharedEvent setSignaledValue:m_submittedCommandBufferCount];
```

Comment:
`m_submittedCommandBufferCount` is incorrect in the general case, because CompositorServices expects us to count from zero